packages/manager/modules/pci/src/projects/onboarding/onboarding.routing.js

```diff
@@ -11,53 +11,56 @@ export default /* @ngInject */ ($stateProvider) => {
       const [$q, publicCloud] = ['$q', 'publicCloud'].map((token) =>
         transition.injector().get(token),
       );
-      return publicCloud
-        .getUnpaidProjects()
-        .then((unPaidProjects) => {
-          if (unPaidProjects.length) {
-            return 'pci.projects';
-          }
-          return $q.all({
-            discoveryProjectId: publicCloud.getDiscoveryProject(),
-            defaultProjectId: publicCloud.getDefaultProject(),
-          });
+      return $q
+        .all({
+          discoveryProjectId: publicCloud.getDiscoveryProject(),
+          defaultProjectId: publicCloud.getDefaultProject(),
+          unPaidProjects: publicCloud.getUnpaidProjects(),
         })
-        .then(({ discoveryProjectId, defaultProjectId }) => {
+        .then(({ discoveryProjectId, defaultProjectId, unPaidProjects }) => {
           const projectId = discoveryProjectId || defaultProjectId;
+
+          if (unPaidProjects.length) {
+            return { state: 'pci.projects' };
+          }
           if (projectId) {
             return { state: 'pci.projects.project', params: { projectId } };
           }
           return false;
         });
     },
     resolve: {
-      goToCreateNewProject: /* @ngInject */ ($state) => () =>
-        $state.go('pci.projects.new'),
-
       goToCreateDiscoveryProject: /* @ngInject */ ($state) => () =>
         $state.go('pci.projects.discovery'),
 
-      model: /* @ngInject */ (pciProjectNew, me) =>
-        pciProjectNew.createOrderCart(me.ovhSubsidiary).then((cart) => {
-          const currentDate = moment().format('YYYY-MM-DD');
-          return {
-            agreements: false,
-            description: `Project ${currentDate}`,
-            cart,
-            hds: cart?.hdsItem !== undefined,
-          };
-        }),
+      isUsRegion: /* @ngInject */ (coreConfig) =>
+        coreConfig.getRegion() === 'US',
```

Comment:
No need to enable it with feature flipping ?

Reply:
Yes